.github/workflows/selenium_tests.yml

```diff
@@ -0,0 +1,80 @@
+name: Selenium Tests
+
+on:
+  pull_request:
+    types: [labeled, synchronize, opened, reopened]
+
+permissions:
+   contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  selenium-sqlite:
+    if: contains(github.event.pull_request.labels.*.name, 'selenium')
+    runs-on: ubuntu-latest
+    name: Selenium tests - SQLite
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+          cache: 'pip'
+          cache-dependency-path: 'tests/requirements/py3.txt'
+
+      - name: Install and upgrade packaging tools
+        run: python -m pip install --upgrade pip setuptools wheel
+      - run: python -m pip install -r tests/requirements/py3.txt .
+
+      - name: Run selenium tests
+        working-directory: ./tests/
+        run: python -Wall runtests.py --verbosity 2 --noinput --selenium=chrome --headles --settings=test_sqlite --parallel 2
+
+  selenium-postgres:
+    if: contains(github.event.pull_request.labels.*.name, 'selenium')
+    runs-on: ubuntu-latest
+    name: Selenium tests - PostgreSQL
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: django
+          POSTGRES_USER: user
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+          cache: 'pip'
+          cache-dependency-path: 'tests/requirements/py3.txt'
+
+      - name: Install and upgrade packaging tools
+        run: python -m pip install --upgrade pip setuptools wheel
+      - run: python -m pip install -r tests/requirements/py3.txt psycopg2 .
+
+      - name: Run selenium tests
+        working-directory: ./tests/
+        run: python -Wall runtests.py --verbosity 2 --noinput --selenium=chrome --headles --settings=test_postgres --parallel 2
+        env:
+          POSTGRES_DB: django
+          POSTGRES_USER: user
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_HOST: localhost
+          POSTGRES_PORT: 5432
```

Comment:
We're using the standard port and hostname, so I'd remove them from environment variables and settings.

Reply:
I tried but I might not have understood what you meant, I was getting connection errors: https://github.com/django/django/actions/runs/5265461513/jobs/9518116077 :thinking: